e2e/tests/functional/plugins/plot/missingPlotObj.e2e.spec.js

```diff
@@ -70,6 +70,8 @@ test.describe('Handle missing object for plots', () => {
 
     //Reloads page and clicks on stacked plot
     await Promise.all([page.reload(), page.waitForLoadState('domcontentloaded')]);
+    await page.reload({ waitUntil: 'domcontentloaded' });
+    await page.goto(stackedPlot.url);
```

Comment:
did you mean to have this reload twice? or is line 72 supposed to be removed

Reply:
I think it was a bad merge